docs/azure/sdk/protocol-convenience-methods.md

```diff
@@ -0,0 +1,209 @@
+{
+    Console.Write($"{item.Category}: ");
+    Console.Write(item.Severity);
+    Console.WriteLine();
+}
+```
+
+The preceding code demonstrates the following `Azure.Core` convenience method patterns:
+
+- Uses a standard C# primitive or model type as a parameter.
+- Returns a friendly C# type that represents the result of the operation.
+
+### Libraries that depend on System.ClientModel
+
+Some Azure client libraries depend directly on the `System.ClientModel` library. For example, the [`OpenAI`](https://www.nuget.org/packages/OpenAI/2.0.0-beta.7) library depends on the `System.ClientModel` library and provides a `ChatClient` class that exposes both protocol and convenience methods.
+
+#### Work with protocol methods
+
+The following code uses a `ChatClient` to call the `CompleteChat` protocol method:
+
+```csharp
+// Create the client
+OpenAIClient client = new("your-openai-key");
+ChatClient chatClient = client.GetChatClient("gpt-4");
+
+// Create the request prompt content
+BinaryData prompt = BinaryData.FromBytes("""
+    {  
+        "model": "gpt-4o",
+        "messages": [
+           {
+               "role": "user",
+               "content": "What is Microsoft Azure?."
+           }
+        ]
+    }
+    """u8.ToArray());
+using BinaryContent content = BinaryContent.Create(prompt);
+
+// Send the request
+ClientResult result = chatClient.CompleteChat(
+        content,
+        new RequestOptions()
+        { 
+            ErrorOptions = ClientErrorBehaviors.NoThrow
+        });
+
+// Retrieve and display the response data
+BinaryData output = result.GetRawResponse().Content;
+using JsonDocument outputAsJson = JsonDocument.Parse(output);
+string message = outputAsJson.RootElement
+    .GetProperty("choices"u8)[0]
+    .GetProperty("message"u8)
+    .GetProperty("content"u8)
+    .GetString();
+
+Console.WriteLine(message);
+```
+
+The preceding code demonstrates the following `System.ClientModel` protocol method patterns:
+
+- Uses the `BinaryContent` type as a parameter to supply data for the request body.
+- Uses the `RequestContext` type to configure request options.
+- Returns data using the `ClientResult` type.
+- Calls the `GetRawResponse` method to access the response data.
+
+#### Work with convenience methods
+
+Consider the following code that uses a `ChatClient` to call the `CompleteChat` convenience method:
+
+```csharp
+// Create the client
+OpenAIClient client = new("your-openai-key");
+ChatClient chatClient = client.GetChatClient("gpt-4");
+
+// Send the prompt
+ClientResult<ChatCompletion> completion
+    = chatClient.CompleteChat("What is Azure?");
+
+// Display the response data
+Console.WriteLine($"{completion.Value.Role}: {completion.Value.Content}");
+```
+
+The preceding code demonstrates the following `System.ClientModel` convenience method patterns:
+
+- Uses a standard C# primitive or model type as a parameter.
+- Returns a `ClientResult` type that represents the result of the operation.
+
```

Comment:
I defer to you @alexwolfmsft, on how deep you want to go and how much detail you want to include per target audience, so feel free to disregard this suggestion if it's not at the level you're thinking.  One thought at this point is that it might be worth pointing out here or elsewhere that from both Azure and SCM-based convenience methods, you can call GetRawResponse on the return type to get access to the HTTP response details.  But that from protocol methods, the Azure clients return the HTTP response details directly, whereas in SCM-based protocol methods, you also call GetRawResponse to get the HTTP response details.  Maybe too low-level or not worth calling attention to -- your call.